apps/files/src/components/FileEntry/FileEntryPreview.vue

```diff
@@ -107,6 +111,7 @@ export default Vue.extend({
 	data() {
 		return {
 			backgroundFailed: undefined as boolean | undefined,
+			backgroundLoaded: false as boolean,
```

Comment:
```suggestion
			backgroundLoaded: false,
```

Not needed as `boolean` is inferred from `false`